cli/formatting.go

```diff
@@ -0,0 +1,192 @@
+		fmt.Sprintf("     limit:|%s", prettifyTime(int64(account.CPULimit.Max))),
+	}
+
+	return columnize.Format(output, config)
+}
+
+func FormatBalances(account *eos.AccountResp, config *columnize.Config) string {
+	totalStaked := account.SelfDelegatedBandwidth.NetWeight.Add(account.SelfDelegatedBandwidth.CPUWeight)
+	totalUnstaking := eos.Asset{
+		Amount: 0,
+		Symbol: account.CoreLiquidBalance.Symbol,
+	}
+	if account.RefundRequest != nil {
+		totalUnstaking = account.RefundRequest.CPUAmount.Add(account.RefundRequest.NetAmount)
+	}
+	total := totalUnstaking.Add(totalStaked).Add(account.CoreLiquidBalance)
+
+	output := []string{
+		fmt.Sprintf("%s balances:", account.CoreLiquidBalance.Symbol.Symbol),
+		fmt.Sprintf("     liquid:|%s", prettifyAsset(account.CoreLiquidBalance)),
+		fmt.Sprintf("     staked:|%s", prettifyAsset(totalStaked)),
+		fmt.Sprintf("     unstaking:|%s", prettifyAsset(totalUnstaking)),
+		fmt.Sprintf("     total:|%s", prettifyAsset(total)),
+	}
+
+	return columnize.Format(output, config)
+}
+
+func FormatProducers(account *eos.AccountResp, config *columnize.Config) string {
+	output := []string{
+		"producers:",
+		fmt.Sprintf("     %s", prettifyAccounts(account.VoterInfo.Producers)),
+	}
+
+	return columnize.Format(output, config)
+}
+
+func prettifyBytes(b int64) string {
+	const u = 1024
+	if b < u {
+		return rightAlignColumnize(fmt.Sprintf("%d", b), "bytes")
+	}
+	div, exp := int64(u), 0
+	for n := b / u; n >= u; n /= u {
+		div *= u
+		exp++
+	}
+	value := float64(b) / float64(div)
+	unit := fmt.Sprintf("%ciB", "KMGTPE"[exp])
+
+	precision := 3
+	if value >= 100 {
+		precision = 1
+	} else if value >= 10 {
+		precision = 2
+	}
+
+	return rightAlignColumnize(strconv.FormatFloat(value, 'f', precision, 64), unit)
+}
+
+func prettifyTime(micro int64) string {
+	value := float64(micro)
+	unit := "μs"
+	if value > 1000000*60*60 {
+		value /= float64(1000000 * 60 * 60)
+		unit = "hr"
+	} else if value > 1000000*60 {
+		value /= float64(1000000 * 60)
+		unit = "min"
+	} else if value > 1000000 {
+		value /= float64(1000000)
+		unit = "sec"
+	} else if value > 1000 {
+		value /= float64(1000)
+		unit = "ms"
+	}
+
+	precision := 3
+	if value >= 100 {
+		precision = 1
+	} else if value >= 10 {
+		precision = 2
+	}
+
+	return rightAlignColumnize(strconv.FormatFloat(value, 'f', precision, 64), unit)
+}
+
+func prettifyAsset(w eos.Asset) string {
+	const unit = 10000
+	formatting := fmt.Sprintf("%%.%df", w.Precision)
+	return rightAlignColumnize(fmt.Sprintf(formatting, float64(w.Amount)/float64(unit)), w.Symbol.Symbol)
+
+}
+
+func prettifyAccounts(accounts []eos.AccountName) string {
+	names := []string{}
+	if len(accounts) == 0 {
+		return "<not voted>"
+	}
+	for _, name := range accounts {
+		names = append(names, string(name))
+	}
+
+	return strings.Join(names, "|  ")
+}
+
+func rightAlignColumnize(value, unit string) string {
+	w := new(tabwriter.Writer)
+	bs := bytes.NewBuffer([]byte(""))

```

Comment:
Could have been `[]byte{}`

Reply:
Yeah... not sure why I did that, will fix

Reply:
Updated